media/json-binding/src/main/java/org/glassfish/jersey/jsonb/internal/JsonBindingProvider.java

```diff
@@ -70,7 +72,24 @@ public Object readFrom(Class<Object> type, Type genericType,
                            MediaType mediaType,
                            MultivaluedMap<String, String> httpHeaders,
                            InputStream entityStream) throws IOException, WebApplicationException {
+        if (entityStream.markSupported()) {
+            entityStream.mark(1);
+            if (entityStream.read() == -1) {
+                throw new NoContentException("JSON-B cannot process empty input stream");
```

Comment:
Please use localization.properties

Reply:
I will do that. Stay tuned.

Reply:
Done with bccd9008528477587b2eb340cbaf0a8762bcdadb.